jdaviz/configs/cubeviz/plugins/sonify_data/sonify_data.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <j-tray-plugin
+    :description="docs_description"
+    :link="docs_link || 'https://jdaviz.readthedocs.io/en/'+vdocs+'/'+config+'/plugins.html#cubeviz-sonify-data'"
+    :uses_active_status="uses_active_status"
+    @plugin-ping="plugin_ping($event)"
+    :keep_active.sync="keep_active"
```

Comment:
instead of using the spectrum at spaxel tool, we could have the mouseover events enabled whenever the plugin is active 🤔 (could be a follow-up, especially given the conversation about potential future audio layers)